packages/experiments/src/components/Pagination/Pagination.styles.ts

```diff
@@ -0,0 +1,59 @@
+import { IPaginationStyles, IPaginationStyleProps } from './Pagination.types';
+import { getGlobalClassNames } from '../../Styling';
+
+const GlobalClassNames = {
+  root: 'ms-Pagination-container',
+  pageNumber: 'ms-Pagination-page-number',
+  selectedPageNumber: 'ms-Pagination-page-number-selected'
+};
+
+export function getStyles(props: IPaginationStyleProps): IPaginationStyles {
+  const { theme } = props;
+
+  const { palette } = theme;
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  return {
+    root: [
+      classNames.root,
+      {
+        listStyle: 'none',
+        selectors: {
+          li: {
+            display: 'inline'
+          }
+        }
+      }
+    ],
+    previousNextPage: {
+      color: palette.black,
+      cursor: 'pointer',
+      border: 'none',
+      backgroundColor: 'Transparent'
+    },
+    disabledPreviousNextPage: {
+      color: palette.neutralTertiaryAlt,
+      border: 'none',
+      backgroundColor: 'Transparent'
+    },
+    pageNumber: [
+      classNames.pageNumber,
+      {
+        color: palette.black,
+        cursor: 'pointer',
+        border: 'none',
+        backgroundColor: 'Transparent'
+      }
+    ],
+    selectedPageNumber: [
+      classNames.selectedPageNumber,
+      {
+        color: palette.blue,
+        fontWeight: 'bold',
+        cursor: 'pointer',
+        border: 'none',
+        backgroundColor: 'Transparent'
```

Comment:
I don't believe `Transparent` is a valid value. Should be `transparent`.

Reply:
'Transparent'  worked. But updated  to be 'transparent'

---
In reply to: [222066908](https://github.com/OfficeDev/office-ui-fabric-react/pull/6532#discussion_r222066908) [](ancestors = 222066908)

Reply:
Addressed.

---
In reply to: [222073340](https://github.com/OfficeDev/office-ui-fabric-react/pull/6532#discussion_r222073340) [](ancestors = 222073340)

Reply:
@caradong make sure 'transparent' color plays well in Safari browser as I had plenty of issues with it. I believe the gist of the issue is that Safari interprets the transparent as transparent-black and it might give some weird colors when a theme is applied.